msg/input_rc.msg

```diff
@@ -22,7 +22,11 @@ uint64 timestamp_last_signal		# last valid reception time
 
 uint8 channel_count			# number of channels actually being seen
 
-int32 rssi				# receive signal strength indicator (RSSI): < 0: Undefined, 0: no signal, 100: full reception
+uint8 RC_RSSI_NO_SIGNAL = 0		# no signal
+uint8 RC_RSSI_MAX = 100			# full reception
+uint8 RC_RSSI_UNDEFINED = 255		# no indication of rssi strength
```

Comment:
This might need to be updated in flight review. https://github.com/PX4/flight_review

Reply:
https://github.com/PX4/flight_review/blob/fe751e4fe71f4a47e7918234265add856ef87631/plot_app/configured_plots.py#L752-L753

Reply:
@dagar, the value on the uorb was 255 on lost, and also now. so there no change need to be done on 'flight review' site.
it might be wrong before, and it wrong now. (it will probably show 2.55 for lost RC) 
but I will fix that later

Reply:
Thanks for looking into it.